core/src/main/java/com/linecorp/armeria/server/http/healthcheck/HttpHealthCheckService.java

```diff
@@ -135,7 +135,10 @@ protected void doGet(ServiceRequestContext ctx, HttpRequest req, HttpResponseWri
         res.respond(newResponse(ctx));
     }
 
-    private AggregatedHttpMessage newResponse(ServiceRequestContext ctx) {
+    /**
+     * Creates a new response which is sent when {@link Server} healthy.
```

Comment:
I realized that it creates both a healthy and unhealthy response. Could you update the Javadoc?

    `Creates a new response which is sent to indicate the healthiness of the {@link Server}.`